inlong-sort/sort-connectors/base/src/test/java/org/apache/inlong/sort/base/dirty/RegexReplaceTest.java

```diff
@@ -32,7 +32,7 @@ public void testRegexReplacement() throws IOException {
         identifier[0] = "yizhouyang";
         identifier[1] = "table2";
         String pattern = "${database}-${table}-${DIRTY_MESSAGE}";
-        String answer = DirtySinkHelper.regexReplace(pattern, DirtyType.BATCH_LOAD_ERROR, "mock message", identifier);
+        String answer = DirtySinkHelper.regexReplace(pattern, DirtyType.BATCH_LOAD_ERROR, "mock message");
```

Comment:
Please fix this UT error

Reply:
@Yizhou-Yang Please check this UT modication

Reply:
Fixed.